Algorithm/QCAlgorithm.History.cs

```diff
@@ -42,7 +42,7 @@ public IHistoryProvider HistoryProvider
         public bool IsWarmingUp
         {
             get;
-            private set;
+            internal set;
```

Comment:
I didn't notice that you have another method to set property. Can be reverted and `SetFinishedWarmingUp()` method called in the test instead.